internal/pgtt_http/http_status.go

```diff
@@ -0,0 +1,46 @@
+package pgtt_http
+
+import (
+	"fmt"
+	"github.com/cybertec-postgresql/pg_timetable/internal/log"
+	"net/http"
+	"sync"
+	"time"
+)
+
+var httpStatus = "starting"
+var httpStatusMutex sync.RWMutex
+
+func setHttpStatus(value string) {
+	httpStatusMutex.Lock()
+	defer httpStatusMutex.Unlock()
+	httpStatus = value
+}
+
+func SetHttpStatusRunning() {
+	setHttpStatus("running")
+}
+
+func StartHTTP(logger log.LoggerHookerIface, portNumber int) {
+	s := &http.Server{
+		Addr: fmt.Sprintf(":%d", portNumber),
+		//Handler:        httpHandler,
+		ReadTimeout:    10 * time.Second,
+		WriteTimeout:   10 * time.Second,
+		MaxHeaderBytes: 1 << 20,
+	}
+	http.HandleFunc("/status", readyHandler)
+	logger.Infof("Starting HTTP server on port %d...", portNumber)
+	err := s.ListenAndServe()
+	if err != nil {
+		logger.WithError(err).Error("Error start HTTP server. %v")
```

Comment:
`Error call has possible formatting directive %v`:
```suggestion
		logger.WithError(err).Error("Cannot start HTTP server")
```